includes/subscriptions/class-wc-payments-subscriptions-plugin-notice-manager.php

```diff
@@ -77,8 +71,16 @@ public function enqueue_scripts_and_styles() {
 	 * Enqueues templates for plugin deactivation warnings on the admin plugin screen.
 	 */
 	public function output_notice_template() {
-		if ( $this->is_admin_plugins_screen() ) {
-			wc_get_template( 'html-subscriptions-plugin-notice.php', [], '', dirname( __DIR__ ) . '/subscriptions/templates/' );
+		if ( ! $this->is_admin_plugins_screen() ) {
+			return;
+		}
+
+		wc_get_template( 'html-subscriptions-plugin-notice.php', [], '', dirname( __DIR__ ) . '/subscriptions/templates/' );
+
+		// Load a slightly different notice for folks still using the legacy WCPay Subscriptions functionality.
+		if ( WC_Payments::get_gateway()->is_subscriptions_plugin_active() ) {
+			wc_get_template( 'html-woo-payments-deactivate-warning.php', [], '', dirname( __DIR__ ) . '/subscriptions/templates/' );
```

Comment:
When I try to test deactivating WooPayments with Woo Subscriptions, the deactivate button doesn't do anything and I can see the following console log:

![image](https://github.com/Automattic/woocommerce-payments/assets/2275145/a020f98d-5d98-4a5e-ac79-2e3d8f247411)

After debugging, I found this file doesn't exist. Do you have it locally?